static/js/containers/ProfileFormContainer.js

```diff
@@ -47,8 +47,10 @@ class ProfileFormContainer extends React.Component {
 
   fetchProfile = () => {
     const { dispatch, profiles, params: { username } } = this.props;
-    if (profiles[username] === undefined || profiles[username].getStatus === undefined) {
-      dispatch(fetchUserProfile(username));
+    let _username = username || SETTINGS.username;
```

Comment:
Maybe we should change `fetchProfile` to take in the username explicitly? Then the calling function can determine which username to use